stdlib/public/runtime/Metadata.cpp

```diff
@@ -392,6 +392,10 @@ static void swift_objc_classCopyFixupHandler(Class oldClass, Class newClass) {
   if (!oldClassMetadata->isTypeMetadata())
    return;
 
+  // rdar://74136916 - Copy the value witness table pointer.
```

Comment:
We don't usually leave comments referencing radars since open source developers can't see them. Can you add a longer explanation? Also, if you're copying it manually here, does it mean it gets copied twice?

Reply:
Yeah, the ObjC runtime copies the entire thing, then calls out to the Swift runtime to fix up any parts it doesn't know how to handle. We then re-copy the ptrauthed pieces. It's kind of gross.

Reply:
I see, thanks for the explanation!

Reply:
Provided a short explanation. The first copy happens in the ObjC runtime via `memcpy`, it can't be avoided.